client/verta/verta/client.py

```diff
@@ -1328,8 +1329,27 @@ def _datasets(self):
         raise NotImplementedError
 
     def _get_dataset_version2(self, id):
+        """
+        Gets a Dataset version.
+
+        Parameters
+        ----------
+        id : str
+            ID of the Dataset version.
+
+        Returns
+        -------
+        :class:`DatasetVersion`
```

Comment:
I think right now we don't have API documentation for DatasetVersion itself, so this might just end up becoming a dead link. What I did earlier was to link it to our current "Dataset versioning" page:
https://github.com/VertaAI/modeldb/blob/master/client/verta/verta/client.py#L764-L766

Reply:
replaced

Reply:
Could you double-check the return types in the docstring of all other methods in this PR too? Thanks!

Reply:
Thank you, replaced